src/components/slider/SliderData.jsx

```diff
@@ -0,0 +1,33 @@
+import ImageOne from './ImgSlider/Card1.png';
+import ImageTwo from './ImgSlider/Card2.png';
+import ImageThree from './ImgSlider/Card3.png';
+import ImageFour from './ImgSlider/Card4.png';
+import ImageFive from './ImgSlider/Card5.png';
+import ImageSix from './ImgSlider/Card6.png';
+import ImageSeven from './ImgSlider/Card7.png';
+
+const SliderData = [
+  {
+    image: ImageOne,
+  },
+  {
+    image: ImageTwo,
+  },
+  {
+    image: ImageThree,
+  },
+  {
+    image: ImageFour,
+  },
+  {
+    image: ImageFive,
+  },
+  {
+    image: ImageSix,
+  },
+  {
+    image: ImageSeven,
+  },
```

Comment:
I don't get why you use objects to store simple data
Why not something like `[imageOne, imageTwo, imageThree, ...]` ? Do you plan on adding more data to these objects ?